x-pack/solutions/observability/plugins/slo/server/services/transform_generators/common.ts

```diff
@@ -16,19 +16,26 @@ import { InvalidTransformError } from '../../errors';
 export function getElasticsearchQueryOrThrow(kuery: QuerySchema = '', dataView?: DataView) {
   try {
     if (kqlQuerySchema.is(kuery)) {
-      return toElasticsearchQuery(fromKueryExpression(kuery));
+      return toElasticsearchQuery(fromKueryExpression(kuery), dataView);
```

Comment:
As we talked during our meeting, do you think it makes sense to use only buildEsQuery?